### ISSUE TITLE
Parse error: syntax error, unexpected '[' /user-login-history/admin/class-user-login-history-table-page.php on line 43

### ISSUE THREAD

**Issue by EmilGH** [faiyazalam/user-login-history#5]
Hello,

After installing User Login History inside WordPress (not from this repository) I receive the following error:

Parse error: syntax error, unexpected '[' in /usr/www/users/hotshoe/wonderlusttravel/wp-content/plugins/user-login-history/admin/class-user-login-history-table-page.php on line 43

I looked at the source and didn't see any obvious errant brackets.  Any suggestions?

Thanks!

Emil

**Comment by faiyazalam** [comment]
@EmilGH I have fixed this bug in this repository. you can download the plugin from this repository and install it. Please let me know if you find any issue. After this I will update the wordpress repository.

What is your php version?

To fix this I just changed the format of array declaration.
Please checkout this topic https://wordpress.org/support/topic/not-able-to-install-due-to-fatal-error/

**Comment by EmilGH** [comment]
Hello!

Thank you for your reply.  Looks like I need to upgrade php.

On the plus side, though, I've downloaded the updated plugin and received a similar error...

Parse error: syntax error, unexpected '[', expecting ')' in /usr/www/users/hotshoe/wonderlusttravel/wp-content/plugins/user-login-history/admin/class-user-login-history-list-table.php on line 37


For what it's worth:

 - I disabled ALL plugins
 - WordPress is 4.8
 - PHP is 5.3.29
 - Apache 2.2.32

Thanks for the assistance!
